charmhelpers/contrib/charmsupport/nrpe.py

```diff
@@ -520,3 +521,29 @@ def remove_deprecated_check(nrpe, deprecated_services):
     for dep_svc in deprecated_services:
         log('Deprecated service: {}'.format(dep_svc))
         nrpe.remove_check(shortname=dep_svc)
+
+
+def add_deferred_restarts_check(nrpe):
```

Comment:
Please could we add docstring comments detailing what the function does, paramters/types, return value/type (if any), and any exceptions that the functions raises?

Reply:
This was an oversight. They are added now, thanks!